diagnosis/README.md

```diff
@@ -19,6 +19,7 @@ The logs retrieved by `getkuberneteslogs.sh` are the following:
 - etcd status and journal
 - docker status and journal
 - kube-system snapshot
+- Kubernetes cloud provider logs
```

Comment:
this is more like `Azure CNI config files`

Reply:
updated